connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -297,7 +297,7 @@ message Expression {
   }
 }
 
-message ScalarInlineUserDefinedFunction {
+message CommonUserDefinedFunction {
```

Comment:
So I'm not sure I 100% agree with the renaming, and would like to invite a bit more feedback from @HyukjinKwon @hvanhovell and others.

The reason why we picked initially `ScalarInlineUserDefinedFunction` was because it represents a scalar function with an inlined defined function body.

If we rename this to `CommonUserDefinedFunction` it's not 100% clear if the message is a scalar function or an aggregate function, does it contain the function body or not.

Personally, if we really have to get rid of the scalar piece, I would prefer `CommonInlineUserDefinedFunction`.

Reply:
Quite valid points, +1 to `CommonInlineUserDefinedFunction`

Reply:
1. Series-to-Scalar Pandas UDFs `groupby(..).agg(..)` should be able to share the same message, so I suggested removing the `Scalar` piece.

2. As for the `Inline` piece, I was thinking we were about to make both registered UDF and unregistered(inline) UDF share the same message so I proposed to remove it.

`CommonInlineUserDefinedFunction` sounds fine. I am just wondering if my understanding of point 2. is correct. @grundprinzip

Reply:
Renamed.